__tests__/e2e/scripts/setup_integration.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+set -e
+
+# Update elastic-package
+go install github.com/elastic/elastic-package@latest
+
+eval "$(elastic-package stack shellinit)"
+
+# Take the stack down
+elastic-package stack down
```

Comment:
Is this a catch flag that takes stack down if the stack was up previously from other tests? 

What happens if one runs setup_integration multiple times? 

Its better to group the setup and test integration together and RIP the stack at one place. The current `setup` and `test` feels a bit out of the way as setup has to be run before the test.

Reply:
`test:integration` actually runs both the test and setup script. This is also how the existing tests work.

This elastic-package stack down is a bit of a fail safe. We actually do run this command after the tests finish as well, but this ensures that you do not have an elastic-package stack up when running locally. This is most helpful for local development, and is pretty irrelevant in the ci context.

Reply:
I actually had multiple problems with the current setup of having setup step and test step where in theory we could merge both and have a single script that effectively handles starting and killing containers on failure. But this is not the scope for this PR, so lets keep whatever way you prefer,